### PR TITLE
Explain how we're going to model field 856 on bib records

### PR DESCRIPTION
**Pull request by alexwlchan**
Rendered version: https://github.com/wellcomecollection/docs/blob/bib-856/rfcs/035-marc-856/README.md

For https://github.com/wellcomecollection/platform/issues/5037

### CODE REVIEW

**Comment by jtweed** [comment]
Looks good to me so far.

File: rfcs/035-marc-856/README.md [hunk 1]
```diff
@@ -0,0 +1,359 @@
...
+*   `url` = the value of subfield $u.
+
+    If subfield $u is skipped or repeated, we will log a warning and not create the item.
+
+    If the contents of subfield $u doesn't look like a URL, we will log a warning and not create the item.
+
+*   `linkText` = the space-joined contents of subfields $z, $y and $3.
+
+    These will be joined in the original subfield order.
+    If this is an empty string, we will omit the linkText field.
```
Comment (alexwlchan) [inline rfcs/035-marc-856/README.md:258]: See discussion in Slack – we don't want this to be arbitrarily long. I'll add some notes from that thread to this PR: https://wellcome.slack.com/archives/C3TQSF63C/p1613990897009600

**Comment by kenoir** [comment]
Really nice clear explanation of quite an opaque topic 👍

**Comment by alexwlchan** [comment]
I'm going to merge this in its current state – it reflects the current implementation of field 856 on bibs.

Please feel free to continue commenting here – I'll get notifications, and then we can iterate on the data we have in the API as necessary.

**Pull request merged**